bin/auth-api/README.md

```diff
@@ -55,9 +27,35 @@ While working on the auth stack, we still need to run it locally and configure t
    ```bash
    pnpm run db:reset
    ```
-5. Navigate to the Tilt dashboard
-6. Enable `auth-api` in the Title dashboard to run it.
-7. `auth-portal` will run automatically once this succeeds.
+5. Tilt dashboard: Enable [`auth-api`](http://localhost:10350/r/auth-api/overview) and then [`auth-portal`](http://localhost:10350/r/auth-portal/overview).
+
+### Prisma / DB
+
+Use `pnpm exec prisma` to run prisma commands locally. For example
```

Comment:
I don't need to use these commands... I use the old structure - why do we need to change away?